homework/index.js

```diff
@@ -30,17 +32,94 @@
     return elem;
   }
 
+  const root = document.getElementById('root');
+  const header = createAndAppend('header', root, { class: 'header' });
+  createAndAppend('p', header, { text: 'HYF Repositories' });
+
+  const select = createAndAppend('select', header, { id: 'selector' });
+  const option = x => {
+    for (let i = 0; i < x.length; i++) {
+      createAndAppend('option', select, { text: x[i].name, value: i });
+    }
+  };
+
+  const div = createAndAppend('div', root, { id: 'container' });
```

Comment:
Hi Ramazan, the name here isn't the best again. If you remember our class from JS1 about naming conventions, variable names should be nouns while function names as verbs describing an action. Please take care of it next time onwards. 🙂

Reply:
I rename it as mainContainer